schematools/cli.py

```diff
@@ -119,12 +120,32 @@ def show_tablenames(db_url):
 )
 @click.argument("dataset_id")
 def show_schema(db_url, dataset_id):
-    """Generate a schema based on an exising relational database."""
+    """Generate a schema based on an existing relational database."""
     engine = _get_engine(db_url)
     aschema = fetch_schema_from_relational_schema(engine, dataset_id)
     click.echo(json.dumps(aschema, indent=2))
 
 
+@show.command("mapfile")
+@click.option(
+    "--schema-url",
+    envvar="SCHEMA_URL",
+    default=DEFAULT_SCHEMA_URL,
+    show_default=True,
+    required=True,
+    help="Url where valid amsterdam schema files are found. "
+    "SCHEMA_URL can also be provided as environment variable.",
+)
```

Comment:
in mijn GeoJSON PR heb ik deze code dry gemaakt. Kunnen we later ook aanpassen :-)